components/Task.js

```diff
@@ -74,6 +75,7 @@ const Task = ({
   const [assignToastOpen, setAssignToastOpen] = useState(false);
   const [unassignToastOpen, setUnassignToastOpen] = useState(false);
   const [isDecided, setIsDecided] = useState(false);
+  const [completedBy, setCompletedBy] = useState(taskCompletedBy);
```

Comment:
State to store who the task is completed by

Reply:
I think this might need to be an array as well.